.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,55 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/python:0-3.10",
+
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/devcontainers/features/git:1": {"version": "os-provided"}
+	},
+
+	// The arguments required when starting in the container.
+	"runArgs": [
+    // "--ipc=host",
+    // "--gpus=all" // Enable GPU
+  ],
```

Comment:
indenting is weird

Reply:
Yes, CPU and GPU are in separate file now